frontend/awx/resources/templates/WorkflowVisualizer/wizard/NodePromptsStep.tsx

```diff
@@ -58,18 +59,17 @@ export function NodePromptsStep() {
       extra_vars: prompt?.extra_vars ?? defaults.extra_vars,
       forks: prompt?.forks ?? defaults.forks,
       instance_groups: prompt?.instance_groups ?? defaults.instance_groups,
-      inventory: prompt?.inventory ?? (defaults.inventory.id ? defaults.inventory : null),
+      inventory: prompt?.inventory ?? defaults.inventory,
       job_slice_count: prompt?.job_slice_count ?? defaults.job_slice_count,
-      job_tags: prompt?.job_tags ?? parseStringToTagArray(defaults.job_tags),
+      job_tags: prompt?.job_tags ?? defaults.job_tags,
       job_type: prompt?.job_type ?? defaults.job_type,
       labels: prompt?.labels ?? readOnlyLabels,
       limit: prompt?.limit ?? defaults.limit,
       organization: prompt?.organization ?? organizationId,
       scm_branch: prompt?.scm_branch ?? defaults.scm_branch,
-      skip_tags: prompt?.skip_tags ?? parseStringToTagArray(defaults.skip_tags),
+      skip_tags: prompt?.skip_tags ?? defaults.skip_tags,
```

Comment:
I don't think we want to change this line.  `parseStringsToArray` puts the string of the skip_tags, which come from the api, into an object so our select component can render them.
```suggestion
      skip_tags: prompt?.skip_tags ?? parseStringToTagArray(defaults.skip_tags),
```